### PR TITLE
Release 4.3

### PR DESCRIPTION
**Pull request by Gustavo-de-Souza**
Adds release-4.3 and changes the order of release-4.2 and release-4.3

### CODE REVIEW

**Review by eduardoconviso — APPROVED** [review]

File: docs/releases/release4.2.md [hunk 1]
```diff
@@ -4,39 +4,71 @@ title: Release 4.2
...
+<div style={{textAlign:'center'}}>
+
+![img](../../static/img/security-feed-4.2.png)
+
+</div>
+
+You are in control of your applications. That was our purpose in building the new Security Feed, to provide you with the vital information and insights you need to make decisions about the security of your products.
+
+- We've added a **Risk Score chart** that provides a holistic view of an asset's risk based on factors such as open vulnerabilities, attack surface, business impact, and data classification. You can also view your highest assets, allowing users to evaluate and prioritize assets with the highest risk scores quickly.
+- In "You might like to know" are, you will receive the news that we have prepared for you and suggestions for some practices you can carry out to improve the security of your applications.
```
Comment (paladini) [inline docs/releases/release4.2.md:34]: Aqui seria ` In "You might like to know" are` seria  `In "You might like to know" area` ?

File: docs/releases/release4.2.md [hunk 2]
```diff
@@ -4,39 +4,71 @@ title: Release 4.2
...
-<div style={{textAlign: 'center'}}>
+<div style={{textAlign:'center'}}>
 
-![img](../../static/img/asset-synchronization-4.2.gif)
+![img](../../static/img/vulnerabity-management-4.2.png)
 
 </div>
 
-Efficiency is at the core of our latest improvements in asset synchronization. We've revamped the synchronization process, boosting its speed by five times compared to the previous version. Now, users can experience faster and more responsive asset updates, ensuring timely awareness of security postures.
+We have made several improvements in the experience related to registering,  managing, and monitoring vulnerability corrections.  These improvements seek to provide greater context in the vulnerabilities identified, group similar vulnerabilities, and facilitate the management process.
```
Comment (paladini) [inline docs/releases/release4.2.md:62]:
```suggestion
We have made several improvements in the experience related to registering, managing, and monitoring vulnerability corrections.  These improvements seek to provide greater context in the vulnerabilities identified, group similar vulnerabilities, and facilitate the management process.
```

Comment (paladini) [inline docs/releases/release4.2.md:62]:
```suggestion
We have made several improvements in the experience related to registering, managing, and monitoring vulnerability corrections.  These improvements seek to provide greater context in the vulnerabilities identified, group similar vulnerabilities, and facilitate the management process.
```

File: docs/releases/release4.2.md (hunk 1 above)
Comment (paladini) [inline docs/releases/release4.2.md:34]:
```suggestion
- In "You might like to know" area, you will receive the news that we have prepared for you and suggestions for some practices you can carry out to improve the security of your applications.
```

**Review by paladini — CHANGES_REQUESTED** [review]

**Pull request merged**